packages/bpk-component-accordion/src/BpkAccordion.js

```diff
@@ -35,9 +35,9 @@ const BpkAccordion = (props: Props) => {
 
   return (
     // $FlowFixMe[cannot-spread-inexact] - inexact rest. See decisions/flowfixme.md
-    <dl className={classNames} {...rest}>
+    <div className={classNames} {...rest}>
```

Comment:
Let me do some checking here for this change, as I think some of these were made for SEO purposes so we may need to do some logic to support both

Reply:
Thanks for doing this. 

If the description elements can't be removed, I would need to chat with the backpack team about the best approach to fix this bug.